source/runbooks/revoking-user-access.html.md.erb

```diff
@@ -0,0 +1,169 @@
+---
+owner_slack: "#modernisation-platform"
+title: Revoking User Access
+last_reviewed_on: 2024-02-12
+review_in: 6 month
+---
+
+# <%= current_page.data.title %>
+
+## Introduction
+
+In the event of a cyber attack or similar malicious activity e.g. some user credentials are compromised, there may be a requirement to revoke user access on the Modernisation Platform (MP).
+
+This page attempts to break down the various users/identities that exist on MP and explain what you would need to do to revoke access.
+
+## Individual IAM Users
+
+### MOJ Member Access
+
+MP members who are MOJ employees access environments using [AWS SSO](https://user-guide.modernisation-platform.service.justice.gov.uk/concepts/environments/single-sign-on.html#single-sign-on)
+which links GitHub Teams to various roles/environments.
+
+Removing user access from GitHub Teams etc. is the responsibility of the relevant application team as explained [here](https://user-guide.modernisation-platform.service.justice.gov.uk/user-guide/our-offer-to-you.html#what-we-expect-from-application-teams),
+however you could contact the [Ops Engineering](https://moj.enterprise.slack.com/archives/C01BUKJSZD4) team who manage our GitHub organisation who will be able to revoke access.
```

Comment:
What roles can be assumed by this type of user (perhaps a link to code, as it's a moving target), what is the session duration for each role (is it specified in code? NOTE, session duration is set per role basis).

How do we revoke current session (NOTE it can be done in the console and probably through aws cli)?

Reply:
ignore this comment, I've noticed you have described roles and revoking session in later points.